python/ray/tune/logger/json.py

```diff
@@ -115,7 +115,9 @@ def log_trial_end(self, trial: "Trial", failed: bool = False):
         del self._trial_files[trial]
 
     def update_config(self, trial: "Trial", config: Dict):
-        self._trial_configs[trial] = config
+        self._trial_configs[trial] = (
```

Comment:
Same here.

Reply:
Right. Makes sense. Thanks for the hint @sven1977!